pyface/ui/qt4/about_dialog.py

```diff
@@ -112,3 +107,22 @@ def _create_contents(self, parent):
         lay.addWidget(buttons)
 
         parent.setLayout(lay)
+
+    def _create_html(self):
+        # Load the image to be displayed in the about box.
+        image = self.image.create_image()
+        path = self.image.absolute_path
+
+        # The additional strings.
+        additions = '<br />'.join(self.additions)
+
+        # Get the version numbers.
+        py_version = platform.python_version()
+        qt_version = QtCore.__version__
+
+        # The additional copyright strings.
+        copyrights = "<br />".join(["Copyright (c) %s" % line
```

Comment:
We should be consistent about using the full `©` symbol.

Reply:
You mean replacing `(c)` with `&copy;`, right?